packages/melody-streams/src/operators/attachEventWithElement.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright 2019 trivago N.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { attachEvent } from './attachEvent';
+import { withElement } from './withElement';
+import { combineRefs } from './combineRefs';
+import { of } from 'rxjs';
+
+export const attachEventWithElement = (...events) => {
```

Comment:
I'm not sure about the ergonomics of this operator. What I don't like is that it produces only a single stream for all of the registered events, forcing the user of the API to do the splitting manually. I have the feeling that it'd be more valuable if it returned one stream per defined event.

The other part of the question is whether we should add this at all. Its a userland solution for an underlying problem: You can only define a single `ref` per element. So if we find a way to add more than one `ref`, that might solve the problem already.

What do you think?